jans-client-api/server/src/test/java/io/jans/ca/server/SpontaneousScopeAuthTest.java

```diff
@@ -1,69 +0,0 @@
-package io.jans.ca.server;
-
-import com.google.common.collect.Lists;
-import io.jans.as.client.AuthorizationResponse;
-import io.jans.ca.client.ClientInterface;
-import io.jans.ca.common.SeleniumTestUtils;
-import io.jans.ca.common.response.RegisterSiteResponse;
-import org.testng.annotations.Parameters;
-import org.testng.annotations.Test;
-
-import java.util.Arrays;
-import java.util.List;
-import java.util.UUID;
-
-import static org.testng.AssertJUnit.*;
-
-public class SpontaneousScopeAuthTest {
```

Comment:
I see many tests are removed (like this one), what is the replacement for them? How are we going to track coverage of existing functionality if we are dropping them now?